source/blender/gpu/intern/gpu_texture.c

```diff
@@ -726,6 +726,11 @@ void GPU_texture_ref(GPUTexture *tex)
 	tex->refcount++;
 }
 
+int GPU_texture_ref_count(GPUTexture *tex)
+{
+	return tex->refcount;
+}
```

Comment:
As GPUMaterial, these classes are defined in the .c and not in the header to obligate the developer to use getter and setter (it make debug easier). See it like a private data of a C++ class.